packages/lambda-functions/event.json

```diff
@@ -0,0 +1,7 @@
+{
+  "Records": [
+    {
+      "body": "{ \"jobId\": \"018a5441-617f-ddea-3d5c-a20c4283d24d\", \"languageCode\": \"es\", \"importLanguage\": \"Spanish\" }"
```

Comment:
This event is used to test the lamdba locally. It mimics the kind of event the lamda would receive from SQS